### PR TITLE
[R2] Example using aws-sdk-ruby

### PR DESCRIPTION
**Pull request by csuhta**
Similar to the other pages, this one provides an example config for Ruby using the official AWS gem

### CODE REVIEW

File: content/r2/examples/aws-sdk-ruby.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+```ruby
+require "aws-sdk-s3"
+
+@r2 = Aws::S3::Client.new(
+  access_key_id: "#{access_key_id}",
+  secret_access_key: "#{secret_access_key}",
+  endpoint: "https://#{cloudflare_account_id}.r2.cloudflarestorage.com",
+  region: "auto",
+  force_path_style: true,
```
Comment (vlovich) [inline content/r2/examples/aws-sdk-ruby.md]: This isn't necessary - we support virtual bucket naming.

**Review by vlovich — APPROVED** [review]
Not a ruby expert. If there's issues I'd prefer to just land this & fixup later as issues are raised.

File: content/r2/examples/aws-sdk-ruby.md (hunk 1 above)
Comment (csuhta) [reply content/r2/examples/aws-sdk-ruby.md]: Ah ok. I copied that from a different doc example. Removed in latest commit.

File: content/r2/examples/aws-sdk-ruby.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+---
+title: Configure `aws-sdk-ruby` for R2
+summary: Example of how to configure `aws-sdk-ruby` to use R2.
+pcx-content-type: configuration
+weight: 1001
+layout: example
+---
+
+You must [generate an Access Key](/r2/platform/s3-compatibility/tokens/) before getting started. All examples will utilities `access_key_id` and `access_key_secret` variables which represent the **Access Key ID** and **Secret Access Key** values you generated. Many Ruby projects also store these credentials in environment variables instead.
```
Comment (deadlypants1973) [inline content/r2/examples/aws-sdk-ruby.md]:
```suggestion
You must [generate an Access Key](/r2/platform/s3-compatibility/tokens/) before getting started. All examples will utilize `access_key_id` and `access_key_secret` variables which represent the **Access Key ID** and **Secret Access Key** values you generated. Many Ruby projects also store these credentials in environment variables instead.
```
I am assuming this was supposed to be a verb?

**Review by deadlypants1973 — CHANGES_REQUESTED** [review]
PCX review, one comment on opening sentence

**Review by deadlypants1973 — APPROVED** [review]

**Pull request merged**